client/src/components/map/MarkerStyles.js

```diff
@@ -1,15 +1,14 @@
-const K_WIDTH = 40;
-const K_HEIGHT = 40;
+const K_WIDTH = 30;
+const K_HEIGHT = 30;
 
 const greatPlaceStyle = {
-  // initially any map object has left top corner at lat lng coordinates
-  // it's on you to set object origin to 0,0 coordinates
+  // // initially any map object has left top corner at lat lng coordinates
+  // // it's on you to set object origin to 0,0 coordinates
   position: 'absolute',
   width: K_WIDTH,
   height: K_HEIGHT,
   left: -K_WIDTH / 2,
   top: -K_HEIGHT / 2,
```

Comment:
Lines 5-11, if i'm reading this correctly... the image has top-left matching the coord, and we shift it up and left by half the width and height so the image is centered on the coords. If we shift up by full height, that would line the coords up with the middle bottom of the image? ie pin point to coords?? But I've tried changing it and my pin doesn't move at all so big shrug